x/gov/spec/07_client.md

```diff
@@ -135,10 +135,14 @@ simd query gov proposal 1
 Example Output:
 
 ```bash
-content:
-  '@type': /cosmos.gov.v1beta1.TextProposal
-  description: testing, testing, 1, 2, 3
-  title: Test Proposal
+messages: [
+  {
+    '@type': /cosmos.gov.v1beta2.MsgVote
+    proposal_id: 10,
+    voter: cosmos1r0tllwu5c9dtgwg3wr28lpvf76hg85f5zmh9l2,
```

Comment:
does this represent a Proposal to vote on another Proposal?

🤯

Reply:
maybe we should use a simpler example, `MsgSend` for instance?

Reply:
Lol okay yeah I can change the example